src/main/webapp/app/exercises/quiz/manage/drag-and-drop-question/drag-and-drop-question-edit.component.html

```diff
@@ -330,11 +330,11 @@ <h3 class="mb-0"><span class="badge bg-warning align-text-top">DnD</span></h3>
         <ng-container *ngIf="!reEvaluationInProgress">
             <hr />
             <div class="question-options">
-                <div class="form-group">
-                    <div class="btn btn-outline-secondary" (click)="togglePreview()">
+                <div ngbTooltip="{{ 'artemisApp.dragAndDropQuestion.disabledPreviewTooltip' | artemisTranslate }}" [disableTooltip]="question.backgroundFilePath !== undefined">
```

Comment:
```suggestion
                <div ngbTooltip="{{ 'artemisApp.dragAndDropQuestion.disabledPreviewTooltip' | artemisTranslate }}" [disableTooltip]="question.backgroundFilePath">
```
Should also work as far as i know :thinking:

Reply:
Yes you are right - sorry for that: Tested locally and `[disableTooltip]="!!question.backgroundFilePath"` works as expected :+1: in the following line you can replace `[disabled]="question.backgroundFilePath === undefined"` with `[disabled]="!question.backgroundFilePath"`